canvas_sdk/methods/accounts.py

```diff
@@ -19,8 +21,10 @@ def list_accounts(request_ctx, per_page=None, **request_kwargs):
         per_page = request_ctx.per_page
     path = '/v1/accounts'
     payload = {
-        'per_page' : per_page,
+        'per_page': per_page,
     }
+    if as_user_id:
+        payload['as_user_id'] = as_user_id
```

Comment:
Will the underlying code not include 'as_user_id as a query parameter if it is set to None? The Canvas API endpoint will choke if as_user_id=None is included in the request.